add_tokens.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+def load_data():
+    try:
+        data = pd.read_csv('additiona_tokens.csv',index_col=[0])
+    except:
+        print('no data found')
+        data = pd.DataFrame()
+    return data
+
+def get_additional_tokens(dex):
+    data = load_data()
+    to_add = pd.Series(dtype=float)
+    if dex in data.index:
+        for i in data.loc[dex]:
+            j = i +'<>weth'
```

Comment:
So my understanding is we have a fixed list of extra tokens and just want to ensure that `pepe<>weth` is in the list somewhere? And maybe we can't just append (except you do that later) because sometimes it's already in our dataframe? I'm having a little trouble visualizing what the before/after `add_tokens` is so I'm having a little trouble simplifying.

Are all the `"X<>Y"` pairs just meant to be a multi-index?

Reply:
X<>Y is simply the way I chose to represent a pool (tokens x, y). So in order to include a new token, we need to insert a pool and later there's logic converting a pool back to the token to track. That's why for all user requests tokens, the code adds a token<>weth pool to the table.